.github/workflows/ci.yml

```diff
@@ -70,13 +74,20 @@ jobs:
       - uses: actions/checkout@v2
 
       - name: Set up Node.js
-        uses: actions/setup-node@v1
+        uses: actions/setup-node@v2.1.4
```

Comment:
do we need the specific version or can we get away with `@v2`? (same with all setup-node steps below)

Reply:
Dependabot, currently, tries to specify GitHub Actions down to the patch version as shown.

Reply:
That's the argument I see in favor of specifying patch version. FWIW, I lean towards just `v2`, but I'm pretty ambivalent on this change.

Reply:
Yep. I just did it because I knew that if I didn't, Dependabot would.